projects/upb/project.yaml

```diff
@@ -0,0 +1,4 @@
+homepage: "<https://github.com/protocolbuffers/upb>"
```

Comment:
I don't think homepage and primary_contact should be in angle brackets.
This is probably why no one was CCed on the [build failure](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=15592).